.circleci/config.yml

```diff
@@ -56,3 +83,8 @@ workflows:
       - build:
           requires:
             - install
+      - docker:
+          context: docker-credentials
+          requires:
+            - build
```

Comment:
I think to require `install` should be enough. The `docker` job does also build the project, but slightly different. Their shared requirement should be the installation part.

Reply:
changed, thanks.